src/smart-escrow/SmartEscrow.sol

```diff
@@ -236,18 +253,30 @@ contract SmartEscrow is AccessControlDefaultAdminRules {
     /// @notice Calculates the amount of OP that has already vested.
     /// @param _timestamp The timestamp to at which to get the vested amount
     function vestedAmount(uint256 _timestamp) public view returns (uint256) {
-        return _vestingSchedule(_timestamp);
+        if (_timestamp > end) {
+            return OP_TOKEN.balanceOf(address(this)) + released;
+        } else {
+            return _vestingSchedule(_timestamp) + _cliffRelease(_timestamp);
+        }
     }
 
     /// @notice Returns the amount vested as a function of time.
     /// @param _timestamp The timestamp to at which to get the vested amount
     function _vestingSchedule(uint256 _timestamp) internal view returns (uint256) {
         if (_timestamp < start) {
             return 0;
-        } else if (_timestamp > end) {
-            return OP_TOKEN.balanceOf(address(this)) + released;
         } else {
-            return initialTokens + ((_timestamp - start) / vestingPeriod) * vestingEventTokens;
+            return ((_timestamp - start) / vestingPeriod) * vestingEventTokens;
```

Comment:
discussed offline - can make diff smaller by just changing original implementation to 
```
return _cliffRelease(_timestamp) + ((_timestamp - start) / vestingPeriod) * vestingEventTokens;
```

Reply:
Implemented.